sources/PseudoTerminal.m

```diff
@@ -6222,7 +6232,7 @@ - (void)safelySetSessionSize:(PTYSession*)aSession rows:(int)rows columns:(int)c
     }
     PtyLog(@"safelySetSessionSize");
     BOOL hasScrollbar = [self scrollbarShouldBeVisible];
-    if (windowType_ == WINDOW_TYPE_NORMAL || windowType_ == WINDOW_TYPE_NO_TITLE_BAR) {
+    if (windowType_ == WINDOW_TYPE_NORMAL || windowType_ == WINDOW_TYPE_NO_TITLE_BAR || windowType_ == WINDOW_TYPE_TOP) {
```

Comment:
Thx for Fix it! :+1: 
I pulled a master(75da04e5c55e68dfa128bf5fab48af0e9374ddc0)